From f3e18ebbf013c59bd8639a6c39eabbe07285871c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Krzysztof=20Drobi=C5=84ski?= <krzysztof@kd-solutions.pl>
Date: Wed, 13 May 2026 10:58:36 +0200
Subject: [PATCH] Adapt Redmine Gemfile for Yocto

Upstream-Status: Inappropriate [Yocto specific]
---
 Gemfile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Gemfile b/Gemfile
index 1f5ee8518..b239635c1 100644
--- a/Gemfile
+++ b/Gemfile
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 ruby '>= 3.2.0', '< 4.1.0'
 
 gem 'rails', '8.1.3.1'
+gem 'puma', '~> 8.0.1'
 gem 'rouge', '~> 5.0'
 gem 'mini_mime', '~> 1.1.0'
 gem "actionpack-xml_parser"
@@ -94,6 +95,7 @@ else
   warn("Please configure your config/database.yml first")
 end
 
+unless ENV["REDMINE_SKIP_DEVELOPMENT_TEST_GEMS"] == "1"
 group :development, :test do
   gem 'debug'
 end
@@ -124,6 +126,8 @@ group :test do
   gem 'webrick'
 end
 
+end
+
 local_gemfile = File.join(File.dirname(__FILE__), "Gemfile.local")
 if File.exist?(local_gemfile)
   eval_gemfile local_gemfile
-- 
2.51.0
